.circleci/config.yml

```diff
@@ -736,7 +736,7 @@ workflows:
             - docker-hub-access
             - citizenlab-ee-environment
           requires:
-            - back-test
+            - back-build-docker-image
```

Comment:
This means that the api docs may be generated, even if the test fails. Do you know what would happen if an acceptance test fails? For example, maybe someone made a mistake causing the expected response to change. I wonder if the docs would then get updated with the wrong response body. But fine for me in any case.

Reply:
Hmm, I assumed the docs will still be generated, but you're right. I should test this.

This would mean that some documentation could be wrong, but I think the impact would be minimal on anyone using the docs.

Reply:
I agree, I think this is acceptable. As long as the documentation doesn't get removed when there are failures, I think this is fine.

Reply:
@sebastienhoorens 

Testing by breaking an acceptance spec = new docs are not produced, as the docs generation step exits with non-zero exit-code = docs remain, but are outdated until the acceptance tests pass again and the docs can be regenerated.

This seems acceptable to me, especially since the failing acceptance test(s) will also make `back-test` fail, which will prompt devs to fix them.

Reply:
Sounds good to me :)